Cargo.toml

```diff
@@ -6,5 +6,7 @@ members = [
 	"fuzz/*",
 	"modules/*",
 	"primitives/*",
+	"primitives/chains/*",
 	"relays/*",
 ]
+exclude = ["primitives/chains"]
```

Comment:
I'm not familiar with `exclude`, so let me know if my understanding here is off: is it basically like if we were to publish the bridges code to `crates.io` that `primitives/chains/*` crates should be excluded from that?

If so, what's the point of doing that?

Reply:
No, it's not related to publishing at all.

In `members` we specify `primitives/*` - i.e. all sub-folders of `primitives` are expected to be workspace members. Cargo is dumb enough, to fail in case `Cargo.toml` file is missing in one of the directories (it makes sense for explicit deps, but not that much for globs), so I have to `exclude` `primitives/chains` from being the workspace member.